omnigibson/objects/controllable_object.py

```diff
@@ -693,12 +702,21 @@ def dump_action(self):
         """
         return self._last_action
 
+    def set_position_orientation(self, position=None, orientation=None, frame: Literal["world", "scene"] = "world"):
```

Comment:
what about `set_linear_velocity`, `set_angular_velocity`, `set_joint_velocity`?
The following might break?
```
desired_lin_vel = ...
obj.set_linear_velocity(desired_lin_velocity)
assert th.isclose(obj.get_linear_velocity(), desired_lin_vel)
```

Reply:
This actually won't crash -- the default behavior is always to use `estimate=False` -- we only use `estimate=True` explicitly when constructing our control dict, so the changes are isolated there